webapp/channels/src/actions/views/profile_popover.ts

```diff
@@ -4,8 +4,10 @@
 import {getTeamMember} from 'mattermost-redux/actions/teams';
 import {getChannelMember} from 'mattermost-redux/actions/channels';
 
-export function getMembershipForEntities(teamId, userId, channelId) {
-    return async (dispatch) => {
+import {DispatchFunc} from 'mattermost-redux/types/actions';
+
+export function getMembershipForEntities(teamId: string, userId: string, channelId?: string) {
```

Comment:
nothing to be change in this one. But I usually tie these types all the way back to the original type like `Team['id']`